md/admin-application-process-list.md

```diff
@@ -0,0 +1,215 @@
+    1. Go to _BPM_>_Processes_ 
+    2. In the row of the process, click on the "View more details" icon. The Process managers currently assigned to the process are shown in the _Process manager mapping_ table.
+    4. To add a user, click the _Plus_ sign or _Pencil_ button in the _User_ column. A popup shows a list of the users who are currently process managers for this process.
+    5. Click _Select users_. A list of users with the Process manager profile is displayed.
+    6. Select the user you want to be a Process manager for the process, and click on _Apply_. 
+     The selected user is added as a Process manager for the process.
+     You can also assign a process to a group, role, or memberships.
+
+## Modify a parameter
+1. Go to _BPM_>_Processes_ 
+2. In the row of the process, click on the "View more details" icon
+3. Click on _Parameters_ in the left menu
+4. In the _Value_ column, click on the value you want to edit. A field appears.
+5. Click on the _Tick_ button to validate your change or _X_ to dismiss your change.
+
+## Edit a connector implementation
+1. Go to _BPM_>_Processes_ 
+2. In the row of the process, click on the "View more details" icon
+3. Click on _Connectors_ in the left menu
+4. In the connector definitions table, in the actions column, click on the _Pencil_ icon
+5. Browse to a .zip file containing the new connector implementation
+6. Click on _Save_ to import the new implementation.
+
+## Upload a new form
+1. Go to _BPM_>_Processes_ 
+2. In the row of the process, click on the "View more details" icon
+3. Click on _Forms_ in the left menu
+4. Click on the _Form list_ tab. A list of form is displayed. These forms are only visible to the current process. 
+5. Click the _plus_ button at the bottom of the list. A file selector popup is displayed. 
+6. Browse to a .zip containing a form.
+7. Click on _Next_, then if your form requires some authorization, it will be displayed.
+8. Click on _Confirm_.
+
+## Edit an existing form
+1. Go to _BPM_>_Processes_ 
+2. In the row of the process, click on the "View more details" icon
+3. Click on _Forms_ in the left menu
+4. Click on the _Form list_ tab. A list of forms is displayed. These forms are only visible to the current process.
+5. Click the _Pencil_ button of a form line. A file selector popup is displayed.
+6. Browse to a .zip containing a form
+7. Click _Next_, then if your form requires some authorization, it will be displayed.
+8. Click on _Confirm_.
+
+## Fix forms
+1. Go to _BPM_>_Processes_ 
+2. In the row of the process, click on the "View more details" icon
+3. Click on _Forms_ in the left menu.
+5. The tables are displayed which list available forms. Click on a red link. A field input appears.
+6. Starting to type some text, and any matching names of installed forms will be proposed. If there is no match for the name you enter, it will be considered as a URL.
+7. Click the _Tick_ button to validate your change or _**X**_ to dismiss your change.
```

Comment:
```suggestion
4. The tables are displayed which list available forms. Click on a red link. A field input appears.
5. Starting to type some text, and any matching names of installed forms will be proposed. If there is no match for the name you enter, it will be considered as a URL.
6. Click the _Tick_ button to validate your change or _**X**_ to dismiss your change.
```